website/docs/networking/vpc-lattice/setup.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "VPC Lattice Setup"
+sidebar_position: 10
+---
+
+# Deploying the AWS Gateway API Controller
```

Comment:
In every other module, we pre-deploy all dependencies to avoid wasting time during in-person labs. However with upcoming changes by @niallthomson to how we are using Terraform and Lattice being a new service, I am okay with including this manual setup flow. I expect most users will want to see the install steps, instead of us abstracting it away from them.

Reply:
ack